<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="ShowImageLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Type ID wanna register...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="verifyButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Verify</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Type name wanna verify...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>30</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
